src/App.js

```diff
@@ -19,7 +20,9 @@ const App = () => {
           path="/routes/ProductsPage/ProductsPage"
           element={<ProductsPage />}
         />
-        <Route path="/" element={<Home />} />
+        <Route path="/routes/Wishlist/Wishlist" element={<Wishlist />} />
+        <Route path="/routes/SignUp/SignUp" element={<SignUp />} />
+        <Route path="/routes/Login/Login" element={<Login />} />
```

Comment:
I have a question. Why are paths like this? You could've just given the path as `/routes/wishlist` or `/wishlist`. Is there any reason behind that?

Reply:
I don't know/wishlist would work.
🧐